scripts/packaging/createbinariesfordocs.ps1

```diff
@@ -90,6 +90,9 @@ try {
     Move-Item "..\MSBuild\Publish\InEditor\WSA\*.dll" "MRTK_$($Version)\dependencies\"
     Copy-Item "$($UnityDirectory)\Data\Managed\UnityEditor.dll" "MRTK_$($Version)\dependencies\"
     Copy-Item "$($UnityDirectory)\Data\Managed\UnityEngine.dll" "MRTK_$($Version)\dependencies\"
+    if ($UnityDirectory.Contains("2018")) {
```

Comment:
Not needed for 2019 or 2020?

Reply:
That file is not needed by either of those versions.